tensorflow_data_validation/coders/tf_example_decoder_test.py

```diff
@@ -17,9 +17,12 @@
 from __future__ import print_function
 
 from absl.testing import absltest
+import apache_beam as beam
+from apache_beam.testing import util
 import numpy as np
 import tensorflow as tf
 from tensorflow_data_validation.coders import tf_example_decoder
+from tensorflow_data_validation.coders.decoder_test_util import _make_example_dict_equal_fn
```

Comment:
Can you import the module name instead of a specific method (to be consistent with the existing practice)?

```suggestion
from tensorflow_data_validation.utils import test_util
```